syne_tune/backend/__init__.py

```diff
@@ -10,8 +10,10 @@
 # on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
 # express or implied. See the License for the specific language governing
 # permissions and limitations under the License.
-__all__ = ['LocalBackend', 'SageMakerBackend']
 
-from syne_tune.backend.local_backend import LocalBackend
-from syne_tune.backend.sagemaker_backend.sagemaker_backend import SageMakerBackend
+__all__ = ['LocalBackend',
```

Comment:
This is only changing code formatting? I am asking because the PR titles mentions "simplify imports for backend" but I do not understand what would be the change.

Reply:
Nothing changes for local and sagemaker, but for simulated there is a simplification.